src/utils/vscodeUtils.ts

```diff
@@ -89,3 +90,18 @@ async function workingFolderInitialized() : Promise<boolean> {
     const workingFolderPath = getDefaultWorkspacePath();
     return await fse.pathExists(path.join(workingFolderPath, `${extensionName}.csproj`));
 }
+
+export async function askFolder(folderLabel: string): Promise<Uri[]> {
```

Comment:
look like we need to return single Uri here

Reply:
Yes - I have reworked this to only return a single Uri